scripts/training.py

```diff
@@ -12,7 +12,7 @@
     parser.add_argument('-l2', '--regularisation_rate', type=float)
     parser.add_argument('-p', '--early_stopping_patience', type=int)
     parser.add_argument('-d', '--early_stopping_delta', type=float)
-    parser.add_argument('-ir', '--imgage_resize', type=int, nargs=2, help='Tuple (width, height) with determines the shape of the resized images')
+    parser.add_argument('-ir', '--imgage_resize', type=int, nargs=2, help='Tuple (width, height) with determines the shape of the resized images. The networks "PreDogNN" and "PreBigDogNN" require a image size of 224x244')
```

Comment:
Update: If I use two Epochs, the same error occurs.

Reply:
> Why do these both networks require this specific images size? And how did you find out?

Because the convolutional network used here is only able to perform on a given image size. 
They will print a message after you load the network if you did not define a specific image size.
However, I think I have to change the image size to the minimum of the respective dataset. 
If I remember correctly, I did it for the training of these networks. 

> Cant we resize batchwise anymore for these two architectures?

Nope, we cant, therefore I added an if condition to the dataloader:
https://github.com/beckstev/MachineLearningSeminar/blob/e4112e02a92e8fbd27394f65b4210fcbcb1eed3a/dog_classifier/net/dataloader.py#L116-L119